### ISSUE TITLE
Earthquake container does not build correctly.

### ISSUE THREAD

**Issue by mohammedzee1000** [CentOS/CentOS-Dockerfiles#88]
While attempting to build the earthquake/centos7 container for the purpose of adding it to the Container Pipeline, i found that the earthquake container does not build. It fails out at step 5 during a build.

Command : 
`$ docker build -t earthquake:centos7 .`

Result :

```
Sending build context to Docker daemon 23.04 kB
Step 1 : FROM centos:centos7
Trying to pull repository registry.centos.org/centos ... 
Pulling repository registry.centos.org/centos
Error: image centos not found
Trying to pull repository docker.io/library/centos ... 
centos7: Pulling from docker.io/library/centos

Digest: sha256:2ae0d2c881c7123870114fb9cc7afabd1e31f9888dac8286884f6cf59373ed9b
Status: Downloaded newer image for docker.io/centos:centos7
 ---> 980e0e4c79ec
........
Step 5 : RUN gem install earthquake
 ---> Running in be56ee31ee6b
Building native extensions.  This could take a while...
Successfully installed eventmachine-1.2.0.1
Successfully installed simple_oauth-0.1.9
Building native extensions.  This could take a while...
ERROR:  Error installing earthquake:
    activesupport requires Ruby version >= 2.2.2.
Successfully installed http_parser.rb-0.5.3
Successfully installed twitter-stream-0.1.16
Successfully installed notify-0.5.2
Successfully installed i18n-0.7.0
Successfully installed thread_safe-0.3.5
Successfully installed tzinfo-1.2.2
Successfully installed minitest-5.9.1
Successfully installed concurrent-ruby-1.0.2
The command '/bin/sh -c gem install earthquake' returned a non-zero code: 1
```

**Comment by mohammedzee1000** [comment]
@jperrin ping

**Comment by mohammedzee1000** [comment]
@jimperrin what about this container?

**Comment by mohammedzee1000** [comment]
ping @jperrin

**Comment by jperrin** [comment]
It looks like there's a dep issue, where this new gem wants ruby 2.2.2 or newer. I'll see about how we might work around this

**Comment by aairey** [comment]
The [earthquake](https://rubygems.org/gems/earthquake/versions/1.0.2) depends on activesupport >= 0.

[activesupport 5.0.1](https://rubygems.org/gems/activesupport/versions/5.0.0.1) depends on ruby >= 2.2.2

But I bet earthquake works with [activesupport 4.2.7.1](https://rubygems.org/gems/activesupport/versions/4.2.7.1) too if you install that specific gem version first.

**TL;DR** The required version by earthquake of activesupport is basicly 'latest', which requires ruby v2.2.2.

**Comment by jperrin** [comment]
Correct. It works specifying 4.2.7.1, so I'll push a commit for that shortly

**Comment by jperrin** [comment]
Updated in the last commit, closing.
